xml/System.Web.Services.Protocols/SoapException.xml

```diff
@@ -284,7 +285,6 @@
       <Docs>
         <param name="message">A message that identifies the reason the exception occurred. This parameter sets the <see cref="P:System.Exception.Message" /> property.</param>
         <param name="code">An <see cref="T:System.Xml.XmlQualifiedName" /> that specifies the type of error that occurred. This parameter sets the <see cref="P:System.Web.Services.Protocols.SoapException.Code" /> property.</param>
-        <param name="subCode">An optional subcode for the SOAP fault. This parameter sets the <see cref="P:System.Web.Services.Protocols.SoapException.SubCode" /> property.</param>
```

Comment:
It seems that most of the params I have to recover every time are still being dropped 
https://github.com/dotnet/dotnet-api-docs/pull/2710/commits/87e454beff958578baea4c5781c30e5fee31570c

Reply:
This issue will be resolved with this PR:
https://github.com/docascode/tripled/pull/1

Reply:
I'm glad we finally figured out the mystery!